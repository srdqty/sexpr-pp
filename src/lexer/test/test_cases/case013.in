; a comment #|
#| another comment
  #|#| nested |#|# |||
  |abc | #

  |#
#;#;(a b c . d)
!#$%^&*asymbol~.#(){}[];a comment #|

; adskjfakjflkj 
+12_98___76_5_4_321__0__(
-12_98___76_5_4_321__0__)
+_12_98___76_5_4_321__0__{
-_12_98___76_5_4_321__0__}

; line comment
+_[
-_]
_ ++12_98___76_5_4_321__0_"abc"_
--12_98___76_5_4_321__0__;#|ok#|
_12_98___76_5_4_321__0__
+_____123ab
123_456_abc
1234567890abc
...#
 
#\a(#\b)#\tab{#\newline}#\space[
#\ ]whatever)
#\#;abc
#\;"def"
#\@[
#\(]#\)(#\[)#\]{#\{}#\}
#\-; comment
= + - _
"!@#$%^&*(){}[]~`abcdefghijklmnopqrstuvwxyz\\;\n\t\r\"
ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789_-+="
#\a

,,@abc'45'#\abc'#\^`123#\newline,@,
abc@`@dude #\x34a654 #(#}#
#|
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb
ccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc
space for further expansion******************************************
ddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddd
*********************************************************************
*********************************************************************
*********************************************************************
*********************************************************************
|#
#\b
