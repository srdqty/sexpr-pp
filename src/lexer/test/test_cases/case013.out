#;@(7,1,75,2)
#;@(7,3,77,2)
LPAREN@(7,5,79,1)
NUMBOOLORSYM(a)@(7,6,80,1)
NUMBOOLORSYM(b)@(7,8,82,1)
NUMBOOLORSYM(c)@(7,10,84,1)
DOT@(7,12,86,1)
NUMBOOLORSYM(d)@(7,14,88,1)
RPAREN@(7,15,89,1)
NUMBOOLORSYM(!#$%^&*asymbol~.#)@(8,1,91,17)
LPAREN@(8,18,108,1)
RPAREN@(8,19,109,1)
stdin:8:20: illegal character '{' (code 123)
